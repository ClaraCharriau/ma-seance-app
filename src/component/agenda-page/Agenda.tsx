import style from './Agenda.module.css';
import AgendaScreeningList from './agenda-screening-list/AgendaScreeningList';

const Agenda = () => {
    return (
        <main className={style.mainAgenda}>
            <h2>Agenda</h2>
            <AgendaScreeningList />
        </main>
    );
};

export default Agenda;
