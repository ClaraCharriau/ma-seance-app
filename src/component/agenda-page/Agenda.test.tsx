import { render } from '@testing-library/react';
import Agenda from './Agenda';
import mockNextWeekNextMonthScreenings from '../../mock/users/user-showtimes-next-week-next-month-screenings.json';
import mockPassedAndNextYearScreenings from '../../mock/users/user-showtimes-passed-screening-next-year-screening.json';
import { BrowserRouter } from 'react-router-dom';

describe('Agenda page tests', () => {
    const agendaContext = require('../../context/agenda.context');
    jest.useFakeTimers().setSystemTime(new Date('2024-03-11'));

    it('should render agenda page with next week and next month screenings', () => {
        jest.spyOn(agendaContext, 'useAgendaContext').mockReturnValue({
            screenings: mockNextWeekNextMonthScreenings
        });
        const component = render(
            <BrowserRouter>
                <Agenda />
            </BrowserRouter>
        );

        expect(component.container).toMatchSnapshot();
    });

    it('should render agenda page with passed screenings and next year screenings', () => {
        jest.spyOn(agendaContext, 'useAgendaContext').mockReturnValue({
            screenings: mockPassedAndNextYearScreenings
        });
        const component = render(
            <BrowserRouter>
                <Agenda />
            </BrowserRouter>
        );

        expect(component.container).toMatchSnapshot();
    });

    it('should render agenda page without any screenings', () => {
        jest.spyOn(agendaContext, 'useAgendaContext').mockReturnValue({
            screenings: []
        });
        const component = render(
            <BrowserRouter>
                <Agenda />
            </BrowserRouter>
        );

        expect(component.container).toMatchSnapshot();
    });
});
