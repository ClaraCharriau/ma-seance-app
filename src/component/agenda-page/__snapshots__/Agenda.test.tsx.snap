// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Agenda page tests should render agenda page with next week and next month screenings 1`] = `
<div>
  <main
    class="mainAgenda"
  >
    <h2>
      Agenda
    </h2>
    <section
      class="agendaPageSection"
    >
      <p>
        Cette semaine
      </p>
      <a
        aria-current="page"
        class="showtimeCardWrapper active"
        href="/"
      >
        <p
          class="calendarDate"
        >
          <svg
            class="calendarFrame"
            fill="none"
            viewBox="0 0 49 59"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M37.7211 4.21515V0.921757C37.727 0.732429 37.7481 0.68071 37.7896 0.58189C37.9722 0.146896 38.6608 -0.00318148 38.7611 5.09622e-05C38.9495 0.0166749 38.9794 0.032837 39.0601 0.0628525C39.4267 0.2 39.663 0.461828 39.6787 0.921757V4.22531C40.0369 4.22716 40.3952 4.22946 40.7534 4.23131C45.0292 4.30612 48.8866 7.78607 48.9689 11.9836C49.0383 22.176 48.9698 40.9333 48.9698 51.1256C48.9419 55.1985 45.292 58.8932 40.7529 58.9726C29.9515 59.034 19.1492 58.973 8.34727 58.973C4.0225 58.9481 0.113721 55.4395 0.0310135 51.2203C-0.0384804 41.0279 0.0305243 22.2706 0.0305243 12.0778C0.0579304 8.00172 3.74795 4.30981 8.24646 4.23131C9.09898 4.22623 9.9515 4.22161 10.8045 4.21746V0.921757C10.8104 0.743511 10.8256 0.713958 10.852 0.636379C11.013 0.169061 11.7427 -0.00318148 11.8445 5.09622e-05C12.0329 0.0166749 12.0627 0.032837 12.1435 0.0628525C12.51 0.2 12.7464 0.461828 12.7621 0.921757V4.20868C21.0813 4.17313 29.4015 4.17544 37.7211 4.21515ZM37.7211 6.06364C29.4015 6.03178 21.0818 6.0641 12.7621 6.07472V10.1573C12.7342 10.9917 12.467 10.9746 11.9663 11.0647C11.4035 11.1658 10.8236 10.7304 10.8045 10.1573V6.07703C9.98723 6.07749 9.16994 6.07796 8.35314 6.07796C5.04093 6.09735 2.05123 8.79736 1.98859 12.0062C1.9186 22.2152 1.9186 40.9887 1.98859 51.1972C2.04878 54.2814 4.85153 57.0654 8.27092 57.1255C19.0899 57.1869 29.9099 57.1869 40.7289 57.1255C44.0025 57.0682 46.9486 54.4061 47.0113 51.1972C47.0813 40.9887 47.0813 22.2152 47.0113 12.0062C46.9511 8.93035 44.1689 6.13845 40.7289 6.07842C40.379 6.07611 40.0286 6.07426 39.6787 6.07241V10.1573C39.6562 10.8186 39.4506 10.8213 39.1712 10.9668C38.5844 11.2711 37.7441 10.8532 37.7211 10.1573V6.06364Z"
              fill="#FAB32D"
            />
          </svg>
          <span>
            dimanche
          </span>
          <span>
            17
          </span>
          <span>
            mars
          </span>
        </p>
        <div
          class="movieTitleAndPlace"
        >
          <p
            class="movieTitle"
          >
            Dune
          </p>
          <div
            class="hourAndPlace"
          >
            <p
              class="hourly"
            >
              11:45
            </p>
            <p
              class="theaterName"
            >
              UGC Ciné Cité Strasbourg Etoile
            </p>
          </div>
        </div>
        <svg
          class="circleSeparator"
          fill="none"
          height="136"
          viewBox="0 0 10 136"
          width="10"
          xmlns="http://www.w3.org/2000/svg"
        >
          <circle
            cx="5"
            cy="5"
            fill="#021F38"
            r="5"
          />
          <circle
            cx="5"
            cy="23"
            fill="#021F38"
            r="5"
          />
          <circle
            cx="5"
            cy="41"
            fill="#021F38"
            r="5"
          />
          <circle
            cx="5"
            cy="59"
            fill="#021F38"
            r="5"
          />
          <circle
            cx="5"
            cy="77"
            fill="#021F38"
            r="5"
          />
          <circle
            cx="5"
            cy="95"
            fill="#021F38"
            r="5"
          />
          <circle
            cx="5"
            cy="113"
            fill="#021F38"
            r="5"
          />
          <circle
            cx="5"
            cy="131"
            fill="#021F38"
            r="5"
          />
        </svg>
        <div
          class="movieCard"
        >
          <a
            class="movieCard"
            href="/movies/cff01b1b-a943-4bcf-a396-afd80417150a/day-1"
          >
            <img
              alt="affiche du film Dune"
              class="movieCardImg"
              src="https://image.tmdb.org/t/p/w500/qpyaW4xUPeIiYA5ckg5zAZFHvsb.jpg"
            />
            <div>
              <p
                class="movieTitle"
              >
                Dune
              </p>
              <p
                class="movieDateDuration"
              >
                31 mars 2024
                 - 
                2h46
              </p>
              <p
                class="movieDirector"
              >
                Réalisé par 
                Denis Villeneuve
              </p>
              <p
                class="movieCast"
              >
                Avec 
                Timothée Chalamet, Zendaya, Rebecca Ferguson
              </p>
            </div>
          </a>
        </div>
      </a>
      <p>
        Ce mois-ci
      </p>
      <a
        aria-current="page"
        class="showtimeCardWrapper active"
        href="/"
      >
        <p
          class="calendarDate"
        >
          <svg
            class="calendarFrame"
            fill="none"
            viewBox="0 0 49 59"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M37.7211 4.21515V0.921757C37.727 0.732429 37.7481 0.68071 37.7896 0.58189C37.9722 0.146896 38.6608 -0.00318148 38.7611 5.09622e-05C38.9495 0.0166749 38.9794 0.032837 39.0601 0.0628525C39.4267 0.2 39.663 0.461828 39.6787 0.921757V4.22531C40.0369 4.22716 40.3952 4.22946 40.7534 4.23131C45.0292 4.30612 48.8866 7.78607 48.9689 11.9836C49.0383 22.176 48.9698 40.9333 48.9698 51.1256C48.9419 55.1985 45.292 58.8932 40.7529 58.9726C29.9515 59.034 19.1492 58.973 8.34727 58.973C4.0225 58.9481 0.113721 55.4395 0.0310135 51.2203C-0.0384804 41.0279 0.0305243 22.2706 0.0305243 12.0778C0.0579304 8.00172 3.74795 4.30981 8.24646 4.23131C9.09898 4.22623 9.9515 4.22161 10.8045 4.21746V0.921757C10.8104 0.743511 10.8256 0.713958 10.852 0.636379C11.013 0.169061 11.7427 -0.00318148 11.8445 5.09622e-05C12.0329 0.0166749 12.0627 0.032837 12.1435 0.0628525C12.51 0.2 12.7464 0.461828 12.7621 0.921757V4.20868C21.0813 4.17313 29.4015 4.17544 37.7211 4.21515ZM37.7211 6.06364C29.4015 6.03178 21.0818 6.0641 12.7621 6.07472V10.1573C12.7342 10.9917 12.467 10.9746 11.9663 11.0647C11.4035 11.1658 10.8236 10.7304 10.8045 10.1573V6.07703C9.98723 6.07749 9.16994 6.07796 8.35314 6.07796C5.04093 6.09735 2.05123 8.79736 1.98859 12.0062C1.9186 22.2152 1.9186 40.9887 1.98859 51.1972C2.04878 54.2814 4.85153 57.0654 8.27092 57.1255C19.0899 57.1869 29.9099 57.1869 40.7289 57.1255C44.0025 57.0682 46.9486 54.4061 47.0113 51.1972C47.0813 40.9887 47.0813 22.2152 47.0113 12.0062C46.9511 8.93035 44.1689 6.13845 40.7289 6.07842C40.379 6.07611 40.0286 6.07426 39.6787 6.07241V10.1573C39.6562 10.8186 39.4506 10.8213 39.1712 10.9668C38.5844 11.2711 37.7441 10.8532 37.7211 10.1573V6.06364Z"
              fill="#FAB32D"
            />
          </svg>
          <span>
            lundi
          </span>
          <span>
            23
          </span>
          <span>
            mars
          </span>
        </p>
        <div
          class="movieTitleAndPlace"
        >
          <p
            class="movieTitle"
          >
            The Grand Budapest Hotel
          </p>
          <div
            class="hourAndPlace"
          >
            <p
              class="hourly"
            >
              18:45
            </p>
            <p
              class="theaterName"
            >
              Pathé Angers
            </p>
          </div>
        </div>
        <svg
          class="circleSeparator"
          fill="none"
          height="136"
          viewBox="0 0 10 136"
          width="10"
          xmlns="http://www.w3.org/2000/svg"
        >
          <circle
            cx="5"
            cy="5"
            fill="#021F38"
            r="5"
          />
          <circle
            cx="5"
            cy="23"
            fill="#021F38"
            r="5"
          />
          <circle
            cx="5"
            cy="41"
            fill="#021F38"
            r="5"
          />
          <circle
            cx="5"
            cy="59"
            fill="#021F38"
            r="5"
          />
          <circle
            cx="5"
            cy="77"
            fill="#021F38"
            r="5"
          />
          <circle
            cx="5"
            cy="95"
            fill="#021F38"
            r="5"
          />
          <circle
            cx="5"
            cy="113"
            fill="#021F38"
            r="5"
          />
          <circle
            cx="5"
            cy="131"
            fill="#021F38"
            r="5"
          />
        </svg>
        <div
          class="movieCard"
        >
          <a
            class="movieCard"
            href="/movies/dd4cdbd2-44b7-4fb4-84b4-4f016d379ddb/day-1"
          >
            <img
              alt="affiche du film The Grand Budapest Hotel"
              class="movieCardImg"
              src="https://image.tmdb.org/t/p/w500/atLMzzA7pOB0BdfM89V7BbdtLN6.jpg"
            />
            <div>
              <p
                class="movieTitle"
              >
                The Grand Budapest Hotel
              </p>
              <p
                class="movieDateDuration"
              >
                26 février 2014
                 - 
                1h40
              </p>
              <p
                class="movieDirector"
              >
                Réalisé par 
                Wes Anderson
              </p>
              <p
                class="movieCast"
              >
                Avec 
                Ralph Fiennes, Zendaya, Rebecca Ferguson
              </p>
            </div>
          </a>
        </div>
      </a>
      <div
        class="pastScreeningsWrapper"
      />
    </section>
  </main>
</div>
`;

exports[`Agenda page tests should render agenda page with passed screenings and next year screenings 1`] = `
<div>
  <main
    class="mainAgenda"
  >
    <h2>
      Agenda
    </h2>
    <section
      class="agendaPageSection"
    >
      <p>
        À venir
      </p>
      <a
        aria-current="page"
        class="showtimeCardWrapper active"
        href="/"
      >
        <p
          class="calendarDate"
        >
          <svg
            class="calendarFrame"
            fill="none"
            viewBox="0 0 49 59"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M37.7211 4.21515V0.921757C37.727 0.732429 37.7481 0.68071 37.7896 0.58189C37.9722 0.146896 38.6608 -0.00318148 38.7611 5.09622e-05C38.9495 0.0166749 38.9794 0.032837 39.0601 0.0628525C39.4267 0.2 39.663 0.461828 39.6787 0.921757V4.22531C40.0369 4.22716 40.3952 4.22946 40.7534 4.23131C45.0292 4.30612 48.8866 7.78607 48.9689 11.9836C49.0383 22.176 48.9698 40.9333 48.9698 51.1256C48.9419 55.1985 45.292 58.8932 40.7529 58.9726C29.9515 59.034 19.1492 58.973 8.34727 58.973C4.0225 58.9481 0.113721 55.4395 0.0310135 51.2203C-0.0384804 41.0279 0.0305243 22.2706 0.0305243 12.0778C0.0579304 8.00172 3.74795 4.30981 8.24646 4.23131C9.09898 4.22623 9.9515 4.22161 10.8045 4.21746V0.921757C10.8104 0.743511 10.8256 0.713958 10.852 0.636379C11.013 0.169061 11.7427 -0.00318148 11.8445 5.09622e-05C12.0329 0.0166749 12.0627 0.032837 12.1435 0.0628525C12.51 0.2 12.7464 0.461828 12.7621 0.921757V4.20868C21.0813 4.17313 29.4015 4.17544 37.7211 4.21515ZM37.7211 6.06364C29.4015 6.03178 21.0818 6.0641 12.7621 6.07472V10.1573C12.7342 10.9917 12.467 10.9746 11.9663 11.0647C11.4035 11.1658 10.8236 10.7304 10.8045 10.1573V6.07703C9.98723 6.07749 9.16994 6.07796 8.35314 6.07796C5.04093 6.09735 2.05123 8.79736 1.98859 12.0062C1.9186 22.2152 1.9186 40.9887 1.98859 51.1972C2.04878 54.2814 4.85153 57.0654 8.27092 57.1255C19.0899 57.1869 29.9099 57.1869 40.7289 57.1255C44.0025 57.0682 46.9486 54.4061 47.0113 51.1972C47.0813 40.9887 47.0813 22.2152 47.0113 12.0062C46.9511 8.93035 44.1689 6.13845 40.7289 6.07842C40.379 6.07611 40.0286 6.07426 39.6787 6.07241V10.1573C39.6562 10.8186 39.4506 10.8213 39.1712 10.9668C38.5844 11.2711 37.7441 10.8532 37.7211 10.1573V6.06364Z"
              fill="#FAB32D"
            />
          </svg>
          <span>
            jeudi
          </span>
          <span>
            10
          </span>
          <span>
            juillet
          </span>
        </p>
        <div
          class="movieTitleAndPlace"
        >
          <p
            class="movieTitle"
          >
            Pauvres créatures
          </p>
          <div
            class="hourAndPlace"
          >
            <p
              class="hourly"
            >
              20:00
            </p>
            <p
              class="theaterName"
            >
              UGC Ciné Cité Strasbourg Etoile
            </p>
          </div>
        </div>
        <svg
          class="circleSeparator"
          fill="none"
          height="136"
          viewBox="0 0 10 136"
          width="10"
          xmlns="http://www.w3.org/2000/svg"
        >
          <circle
            cx="5"
            cy="5"
            fill="#021F38"
            r="5"
          />
          <circle
            cx="5"
            cy="23"
            fill="#021F38"
            r="5"
          />
          <circle
            cx="5"
            cy="41"
            fill="#021F38"
            r="5"
          />
          <circle
            cx="5"
            cy="59"
            fill="#021F38"
            r="5"
          />
          <circle
            cx="5"
            cy="77"
            fill="#021F38"
            r="5"
          />
          <circle
            cx="5"
            cy="95"
            fill="#021F38"
            r="5"
          />
          <circle
            cx="5"
            cy="113"
            fill="#021F38"
            r="5"
          />
          <circle
            cx="5"
            cy="131"
            fill="#021F38"
            r="5"
          />
        </svg>
        <div
          class="movieCard"
        >
          <a
            class="movieCard"
            href="/movies/3d8f1342-15f1-44b1-a48f-4581d654b94a/day-1"
          >
            <img
              alt="affiche du film Pauvres créatures"
              class="movieCardImg"
              src="https://image.tmdb.org/t/p/w500/5fT98da9ccWN2xr8VOJrSBp3Cdw.jpg"
            />
            <div>
              <p
                class="movieTitle"
              >
                Pauvres créatures
              </p>
              <p
                class="movieDateDuration"
              >
                17 janvier 2024
                 - 
                1h30
              </p>
              <p
                class="movieDirector"
              >
                Réalisé par 
                Yorgos Lanthimos
              </p>
              <p
                class="movieCast"
              >
                Avec 
                Emma Stone, Mark Ruffalo, Willem Dafoe
              </p>
            </div>
          </a>
        </div>
      </a>
      <p
        class="pastScreeningsTitle"
      >
        <svg
          fill="none"
          height="19"
          viewBox="0 0 19 19"
          width="19"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M9.5 19C7.62108 19 5.78435 18.4428 4.22209 17.399C2.65982 16.3551 1.44218 14.8714 0.723149 13.1355C0.0041162 11.3996 -0.184015 9.48946 0.182544 7.64665C0.549104 5.80383 1.45389 4.11109 2.78249 2.78249C4.11109 1.45389 5.80383 0.549104 7.64665 0.182544C9.48946 -0.184015 11.3996 0.0041162 13.1355 0.723149C14.8714 1.44218 16.3551 2.65982 17.399 4.22209C18.4428 5.78435 19 7.62108 19 9.5C19 12.0196 17.9991 14.4359 16.2175 16.2175C14.4359 17.9991 12.0196 19 9.5 19ZM9.5 1.46154C7.91015 1.46154 6.35599 1.93299 5.03407 2.81627C3.71216 3.69954 2.68185 4.95498 2.07343 6.42382C1.46502 7.89265 1.30583 9.50892 1.616 11.0682C1.92616 12.6275 2.69175 14.0599 3.81595 15.1841C4.94015 16.3083 6.37247 17.0738 7.93178 17.384C9.49109 17.6942 11.1074 17.535 12.5762 16.9266C14.045 16.3182 15.3005 15.2878 16.1837 13.9659C17.067 12.644 17.5385 11.0899 17.5385 9.5C17.5385 7.36807 16.6916 5.32346 15.1841 3.81595C13.6765 2.30845 11.6319 1.46154 9.5 1.46154Z"
            fill="#FFFDEF"
          />
          <path
            d="M13.5191 14.25C13.4229 14.2505 13.3275 14.2321 13.2385 14.1957C13.1495 14.1594 13.0685 14.1058 13.0002 14.0381L8.98098 10.0188C8.91325 9.95055 8.85967 9.86957 8.8233 9.78053C8.78693 9.6915 8.7685 9.59616 8.76906 9.49999V3.65383C8.76906 3.46002 8.84605 3.27415 8.98309 3.1371C9.12014 3.00006 9.30601 2.92307 9.49982 2.92307C9.69364 2.92307 9.87951 3.00006 10.0166 3.1371C10.1536 3.27415 10.2306 3.46002 10.2306 3.65383V9.20037L14.0379 13.0004C14.1064 13.0683 14.1608 13.1491 14.1979 13.2382C14.235 13.3272 14.2541 13.4227 14.2541 13.5192C14.2541 13.6157 14.235 13.7112 14.1979 13.8003C14.1608 13.8893 14.1064 13.9701 14.0379 14.0381C13.9696 14.1058 13.8886 14.1594 13.7996 14.1957C13.7106 14.2321 13.6152 14.2505 13.5191 14.25Z"
            fill="#FFFDEF"
          />
        </svg>
        Séances passées
      </p>
      <div
        class="pastScreeningsWrapper"
      >
        <a
          aria-current="page"
          class="showtimeCardWrapper active"
          href="/"
        >
          <p
            class="calendarDate"
          >
            <svg
              class="calendarFrame"
              fill="none"
              viewBox="0 0 49 59"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M37.7211 4.21515V0.921757C37.727 0.732429 37.7481 0.68071 37.7896 0.58189C37.9722 0.146896 38.6608 -0.00318148 38.7611 5.09622e-05C38.9495 0.0166749 38.9794 0.032837 39.0601 0.0628525C39.4267 0.2 39.663 0.461828 39.6787 0.921757V4.22531C40.0369 4.22716 40.3952 4.22946 40.7534 4.23131C45.0292 4.30612 48.8866 7.78607 48.9689 11.9836C49.0383 22.176 48.9698 40.9333 48.9698 51.1256C48.9419 55.1985 45.292 58.8932 40.7529 58.9726C29.9515 59.034 19.1492 58.973 8.34727 58.973C4.0225 58.9481 0.113721 55.4395 0.0310135 51.2203C-0.0384804 41.0279 0.0305243 22.2706 0.0305243 12.0778C0.0579304 8.00172 3.74795 4.30981 8.24646 4.23131C9.09898 4.22623 9.9515 4.22161 10.8045 4.21746V0.921757C10.8104 0.743511 10.8256 0.713958 10.852 0.636379C11.013 0.169061 11.7427 -0.00318148 11.8445 5.09622e-05C12.0329 0.0166749 12.0627 0.032837 12.1435 0.0628525C12.51 0.2 12.7464 0.461828 12.7621 0.921757V4.20868C21.0813 4.17313 29.4015 4.17544 37.7211 4.21515ZM37.7211 6.06364C29.4015 6.03178 21.0818 6.0641 12.7621 6.07472V10.1573C12.7342 10.9917 12.467 10.9746 11.9663 11.0647C11.4035 11.1658 10.8236 10.7304 10.8045 10.1573V6.07703C9.98723 6.07749 9.16994 6.07796 8.35314 6.07796C5.04093 6.09735 2.05123 8.79736 1.98859 12.0062C1.9186 22.2152 1.9186 40.9887 1.98859 51.1972C2.04878 54.2814 4.85153 57.0654 8.27092 57.1255C19.0899 57.1869 29.9099 57.1869 40.7289 57.1255C44.0025 57.0682 46.9486 54.4061 47.0113 51.1972C47.0813 40.9887 47.0813 22.2152 47.0113 12.0062C46.9511 8.93035 44.1689 6.13845 40.7289 6.07842C40.379 6.07611 40.0286 6.07426 39.6787 6.07241V10.1573C39.6562 10.8186 39.4506 10.8213 39.1712 10.9668C38.5844 11.2711 37.7441 10.8532 37.7211 10.1573V6.06364Z"
                fill="#FAB32D"
              />
            </svg>
            <span>
              vendredi
            </span>
            <span>
              10
            </span>
            <span>
              janvier
            </span>
          </p>
          <div
            class="movieTitleAndPlace"
          >
            <p
              class="movieTitle"
            >
              Your Name.
            </p>
            <div
              class="hourAndPlace"
            >
              <p
                class="hourly"
              >
                10:45
              </p>
              <p
                class="theaterName"
              >
                Cinéma Vox
              </p>
            </div>
          </div>
          <svg
            class="circleSeparator"
            fill="none"
            height="136"
            viewBox="0 0 10 136"
            width="10"
            xmlns="http://www.w3.org/2000/svg"
          >
            <circle
              cx="5"
              cy="5"
              fill="#021F38"
              r="5"
            />
            <circle
              cx="5"
              cy="23"
              fill="#021F38"
              r="5"
            />
            <circle
              cx="5"
              cy="41"
              fill="#021F38"
              r="5"
            />
            <circle
              cx="5"
              cy="59"
              fill="#021F38"
              r="5"
            />
            <circle
              cx="5"
              cy="77"
              fill="#021F38"
              r="5"
            />
            <circle
              cx="5"
              cy="95"
              fill="#021F38"
              r="5"
            />
            <circle
              cx="5"
              cy="113"
              fill="#021F38"
              r="5"
            />
            <circle
              cx="5"
              cy="131"
              fill="#021F38"
              r="5"
            />
          </svg>
          <div
            class="movieCard"
          >
            <a
              class="movieCard"
              href="/movies/3e07de0f-f2c8-4c46-9ac9-744abb8af481/day-1"
            >
              <img
                alt="affiche du film Your Name."
                class="movieCardImg"
                src="https://image.tmdb.org/t/p/w500/zyHjvVRgKOt9wgVx4ikp2kGArGF.jpg"
              />
              <div>
                <p
                  class="movieTitle"
                >
                  Your Name.
                </p>
                <p
                  class="movieDateDuration"
                >
                  28 décembre 2016
                   - 
                  1h47
                </p>
                <p
                  class="movieDirector"
                >
                  Réalisé par 
                  Makoto Shinkai
                </p>
                <p
                  class="movieCast"
                >
                  Avec 
                  Ryunosuke Kamiki, Mone Kamishiraishi, Ryo Narita
                </p>
              </div>
            </a>
          </div>
        </a>
      </div>
    </section>
  </main>
</div>
`;

exports[`Agenda page tests should render agenda page without any screenings 1`] = `
<div>
  <main
    class="mainAgenda"
  >
    <h2>
      Agenda
    </h2>
    <section
      class="agendaPageSection"
    >
      <div
        class="pastScreeningsWrapper"
      />
      <div
        class="emptyAgendaMessage"
      >
        <p>
          Rien de prévu...
        </p>
        <a
          href="/currently"
        >
          CONSULTEZ LES SORTIES DE LA SEMAINE
        </a>
      </div>
    </section>
  </main>
</div>
`;
